capi/VideoDecoderCapi.h

```diff
@@ -20,8 +20,8 @@
 #ifndef __ENABLE_CAPI__
 #define __ENABLE_CAPI__ 1
 #endif
-#include "interface/VideoDecoderDefs.h"
-#include "interface/VideoCommonDefs.h"
+#include "VideoDecoderDefs.h"
+#include "VideoCommonDefs.h"
```

Comment:
I just looked at libyami-utils, the files VideoDecoderCapi..h and VideoEncoderCapi..h are correct over there :)
Yea, they should be removed.
I see there is another PR for always including capi(which I like), maybe this can be fixed after that is addressed.